### ISSUE TITLE
hotspot is not activating

### ISSUE THREAD

**Issue by Joshaby** [linuxmint/cinnamon-control-center#236]
```
 * 4.6.0-unstable-202005121540~ubuntu20.04.1
 * Mint 20
 * HD 620
 * 64 bit
 ```

**Issue**
![Captura de tela de 2020-06-06 20-19-13](https://user-images.githubusercontent.com/56325844/83956336-57ea6900-a833-11ea-80e7-be47d83b54c3.png)

hotspot is not activating

**Steps to reproduce**

**Expected behaviour**
The hostpot being activated

**Other information**

**Comment by mchilli** [comment]
Same here, you can only start a hotspot if you start cinnamon-settings as root.
Btw is it possible to use the pre configured ssid instead of change it to the hostname?

**Comment by Fantu** [comment]
duplicate of https://github.com/linuxmint/cinnamon-control-center/issues/248 and probably this issue is solved in 4.8.2 (with b225e86 )

**Comment by Joshaby** [comment]
Thank you!
